ivre/utils.py

```diff
@@ -2375,4 +2365,14 @@ def ntlm_extract_info(value):
         return _ntlm_authenticate_info(value)
 
     # NTLM_NEGOTIATE messages are not handled yet
-    return "NTLM"
+    return {}
+
+
+def dict2string(dic):
```

Comment:
The name sounds too generic for what it means. Also, this is clearly an internal function, so maybe we could call it something like this?
```suggestion
 def _ntlm_dict2string(dic): 
```